app/views/categories/index.html.erb

```diff
@@ -23,12 +35,17 @@
 				    	<h5 style="height: 200px;"><%= category.description %></h5>
 				    </div>
 				    <div class="text-center">
-				    	<%= link_to "Let's Start !",  "#", class: "btn btn-outline-primary btn-lg mt-4" %>
-				    	<%= link_to "Review Words",  "#", class: "btn btn-outline-warning btn-lg mt-4 ml-1" %>
-				    </div>
+				    <% @lesson_test = @lesson.find_by(category_id: category.id) %>
+				    <% if  @lesson.pluck(:category_id).any?(category.id) %>
```

Comment:
@nozomi-matsuoka Add `elsif` condition here to check if words is empty
`category.words.empty`